src/pages/SellerListMain/SellerListMain.js

```diff
@@ -1,37 +1,92 @@
 import React, { useState, useEffect, useHistory } from 'react';
-import styled from 'styled-components';
 import SellerList from '../../components/SellerList';
 import SellerTag from '../../components/SellerTag';
 import SellerSearch from '../../components/SellerSearch';
 import SellerProducts from '../../components/SellerProducts';
 import { API } from '../../config';
-import axios from 'axios';
+import axios, { get } from 'axios';
+import InfiniteScroll from 'react-infinite-scroll-component';
 
 const SellerListMain = props => {
   const [sellerList, setSellerList] = useState([]);
   const [productList, setProductList] = useState([]);
+  const [searchWord, setSearchWord] = useState();
+  const [tagInit, setTagInit] = useState('radio1');
 
   useEffect(() => {
-    axios({
-      method: 'get',
-      url: `${API.SELLER}`,
-      responseType: 'stream',
-    })
+    axios
+      .get(API.SELLER)
+
       .then(result => {
-        setSellerList(result.data.seller_list);
+        console.log(result.data);
+        setSellerList(result.data.seller);
       })
       .catch(error => {
         console.log(error.message);
       });
   }, []);
 
-  console.log(sellerList);
+  // const searchReady = keyword => {
+
+  // };
+
+  const searchStart = keyword => {
+    setSearchWord(keyword);
+
+    axios
+      .get(API.SEARCHLST + `?keyword=${keyword}`)
+      .then(result => {
+        console.log(result.data);
+        setSellerList(result.data.seller);
+        setProductList(result.data.item);
+      })
+      .catch(error => {
+        console.log(error.message);
+      });
+  };
+
+  console.log(productList);
+
+  // useEffect(() => {
+  //   axios
+  //     .get(API.PRODUCT)
+  //     .then(result => {
+  //       setProductList(result.data.item);
+  //     })
+  //     .catch(error => {
+  //       console.log(error.message);
+  //     });
+  // }, []);
+
+  const tagFilter = tagId => {
+    setTagInit(tagId);
+    console.log(tagId);
+    axios
+      .get(API.SELLERLST + `?category=${tagId}`)
+      .then(result => {
+        console.log(result.data);
+        setSellerList(result.data.seller);
+        setProductList(result.data.item);
+      })
+      .catch(error => {
+        console.log(error.message);
+      });
+  };
+
+  console.log(searchWord);
   return (
     <>
-      <SellerSearch />
-      <SellerTag />
+      <SellerSearch searchStart={searchStart} />
+      {searchWord === undefined ? (
+        <SellerTag
+          sellerList={sellerList}
+          tagFilter={tagFilter}
+          data={tagInit}
+        />
+      ) : null}
```

Comment:
- 이 경우라면 삼항연산자 보다는 && 연산자가 의미적으로 더 적합합니다. 삼항 연산자는 경우의 수가 2가지인 경우에 사용해주세요!
```suggestion
      {!searchWord && (
        <SellerTag
          sellerList={sellerList}
          tagFilter={tagFilter}
          data={tagInit}
        />
      )}
```